.gitignore

```diff
@@ -1,3 +1,9 @@
 __pycache__
 build
+
+# Python Virtual Environment
+venv/
```

Comment:
Thanks, learned something very interesting!

Reply:
I just tried it out, and of course it works perfectly. But I'm thinking about whether it is generally a good idea to just put them in the public .gitignore. Because there is a (slight) chance that others have similar setups and then they don't forget to add it to their .git/info/exclude and it won't get into the repo.

But if you'd prefer me to it in .git/info/exclude in this case, just tell me. ;) What about the .idea/ in there as well?

Reply:
I don't use a virtual environment for this project, but if I would I wouldn't name it `venv`. 😉 

I think only build artifacts and *maybe* config files that everyone has to use should be in `.gitignore`. Everything else should be added locally.

Reply:
Yes, that sounds fair. I will do that!